mattermost-community/mattermost-plugin-welcomebot
# [MM-387] Add a global welcome message field

#### Summary
- Added a global welcome message field
- Updated Readme with the addition of new field

#### Screenshot
![image](https://github.com/mattermost/mattermost-plugin-welcomebot/assets/100013900/847029e3-17d0-46f4-aa0b-5a6f78b9847a)

#### Checklist

#### Ticket Link
Fixes #36 

#### What to test?
- Add a new field `GlobalWelcomeMessage` in the welcome bot config
- Create a new user and log in the Mattermost server from the new user
- Check for the global message present in the welcome bot DM
- Some edge cases to test: Add only `GlobalWelcomeMessage` in the config, add both `TeamName` and `GlobalWelcomeMessage` in the config
